docs/cmdline-opts/list-only.d

```diff
@@ -2,15 +2,14 @@ c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
 SPDX-License-Identifier: curl
 Long: list-only
 Short: l
-Protocols: FTP POP3
+Protocols: FTP POP3 SFTP
 Help: List only mode
 Added: 4.0
-Category: ftp pop3
+Category: ftp pop3 sftp
 Example: --list-only ftp://example.com/dir/
 See-also: quote request
 Multi: boolean
 ---
-(FTP)
```

Comment:
Actually, I was too early suggesting you remove this line. But just adding SFTP to it is also not enough. The paragraphs below this header explains FTP details that only partly overlaps with how it works for SFTP...

Reply:
I changed the patch splitting the definition between FTP and SFTP.

Reply:
:+1: